model/src/main/java/org/alfresco/transform/config/reader/TransformConfigReaderFactory.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * #%L
+ * Alfresco Transform Model
+ * %%
+ * Copyright (C) 2015 - 2022 Alfresco Software Limited
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.alfresco.transform.config.reader;
+
+import static java.util.Objects.isNull;
+
+import org.springframework.core.io.Resource;
+
+public class TransformConfigReaderFactory
```

Comment:
If this should not be instantiated, which I think is the case. Please provide a private constructor to hide the implicit public one.

Reply:
done